### PR TITLE
Pull #12792: Execute upload of -all jar separately

### PR DESCRIPTION
**Pull request by stoyanK7**
Requires https://github.com/checkstyle/checkstyle/pull/12793 to be merged first.

Success upload run:
https://github.com/stoyanK7/checkstyle/actions/runs/4326378052
image

Proof of upload:
https://github.com/stoyanK7/checkstyle/releases/tag/checkstyle-10.8.1
image

### CODE REVIEW

File: .github/workflows/new-milestone-and-issue-in-other-repo.yml [hunk 1]
```diff
@@ -51,12 +51,6 @@ jobs:
         run: |
           ./.ci/close-create-milestone.sh
 
-      - name: Upload '-all' jar
-        env:
-          GITHUB_TOKEN: ${{ github.token }}
-        run: |
-          ./.ci/upload-all-jar.sh ${{ inputs.version }}
-
```
Comment (stoyanK7) [inline .github/workflows/new-milestone-and-issue-in-other-repo.yml:59]: After merging of PR, https://github.com/checkstyle/checkstyle/wiki/Semi-automated-release has to be updated.

Comment (stoyanK7) [reply .github/workflows/new-milestone-and-issue-in-other-repo.yml:59]: I will open another PR for this actually.

**Review by romani — APPROVED** [review]
Ok to merge

**Comment by rnveach** [comment]
@stoyanK7 let us know when this is no longer a draft.

**Comment by stoyanK7** [comment]
@rnveach @romani Ready for review.

**Review by rnveach — APPROVED** [review]

**Comment by romani** [comment]
@rnveach , @stoyanK7 ,
after merge, action should put milestone on this PR.

**Comment by stoyanK7** [comment]
Milestone will not be put unfortunately..

**Comment by romani** [comment]
I set milestone myself

**Pull request merged**